### PR TITLE
MRG: Fix BEM chdir

### LINKED ISSUE #2598: chdir when watershed
Hi MNEers
I noticed that when I run mne.bem.make_watershed_bem I finish in the subject's directory/watershed
This could be comfortable for certain purposes but I generally find it confusing.
what do you think?

### PR DESCRIPTION
**Pull request by Eric89GXL**
Closes #2598.

### CODE REVIEW

File: mne/bem.py [hunk 1]
```diff
@@ -1048,19 +1048,17 @@ def make_watershed_bem(subject, subjects_dir=None, overwrite=False,
...
-    os.chdir(ws_dir)
     if op.isfile(T1_mgz):
         # XXX : do this with python code
-        surfaces = [subject + '_brain_surface', subject +
-                    '_inner_skull_surface', subject + '_outer_skull_surface',
-                    subject + '_outer_skin_surface']
-        for s in surfaces:
+        surfs = ['brain_surface', 'inner_skull', 'outer_skull', 'outer_skin']
+        for s in surfs:
+            s = op.join(ws_dir, '%s_%s_surface' % (subject, s))
```
Comment (jaeilepp) [inline mne/bem.py]: Wouldn't this make the first one ``brain_surface_surface``?

Comment (agramfort) [inline mne/bem.py]: Indeed

Comment (Eric89GXL) [inline mne/bem.py]: all better

Comment (agramfort) [inline mne/bem.py]: did you push? 

surfs = ['brain_surface', 'inner_skull', 'outer_skull', 'outer_skin']
->
surfs = ['brain', 'inner_skull', 'outer_skull', 'outer_skin']

Comment (Eric89GXL) [inline mne/bem.py]: Yes, GitHub changed the way emails work if you open the link it takes you to the old page, look at the top there is a "You are viewing a subset of the changes. View all changes"

**Comment by agramfort** [comment]
LGTM

did you test it on a subject?

**Comment by Eric89GXL** [comment]
No, I have never really gotten the `test_commands` to work with the BEM test. Or at least it hangs for > 20 minutes and I kill it too early.

**Comment by Eric89GXL** [comment]
(I don't really use this code to do BEMs)

**Comment by Eric89GXL** [comment]
Is there an easy way to test it?

**Comment by agramfort** [comment]
merged by rebase after testing and a fix

thx @Eric89GXL

**Pull request closed without merging**